Update /fleetcarrier CAPI endpoint docs

aussig opened the pull request:
Update to include all latest data, predominantly related to on-foot services and microresources.

aussig committed:
Update /fleetcarrier CAPI endpoint docs

Update to include all latest data, predominantly related to on-foot services and microresources.

Athanasius commented on docs/FrontierDevelopments-CAPI-endpoints.md:
```diff
@@ -416,6 +437,50 @@ provides access to information about the Cmdr's fleet carrier.
+            1. `outstanding`: The remaining outstanding to fulfil the buy order
+            1. `price`: The buy price per item
+    1. `onfootmicroresources`: Buy and sell orders for micro resources (on foot materials)
+        1. `sales`: A Dict containing sell orders, with the key being the sell order ID (str) and the contents being a Dict in the following format:
```
This could use a note about how empty `sales` is a list, although it's an object when it has data.

Athanasius commented on docs/FrontierDevelopments-CAPI-endpoints.md:
```diff
@@ -466,12 +531,18 @@ provides access to information about the Cmdr's fleet carrier.
+
 1. `modules`: Available modules in outfitting
 
+1. `timestamp`: Timestamp of last update in ISO 8601 interchange standard, using `Z` as the UTC timezone suffix.
```
I can't find this in current Live ghalaxy `/fleetcarrier` output.  Is it being added by whatever you're using to query ?

aussig replied:
Mmm, I grabbed this directly from the `CAPIData.data` being passed from the new EDMC code. I can now see that the timestamp is injected into the `CAPIData` object in `companion.py` so that's where it came from.   Will remove.

Athanasius replied:
I wondered if that was the case :) .

aussig committed:
Modification from PR review

Refined /fleetcarrier docs to include empty List / populated Dict options for `orders` -> `onfootmicroresources` -> `sales`

aussig committed:
Further update from PR review

Remove `timestamp` from /fleetcarrier endpoint docs - was being injected by EDMC so not in source response.
